features/document-parsing.mdx

```diff
@@ -22,7 +22,12 @@ Firecrawl currently supports the following document formats:
 - **Word Documents** (`.docx`, `.doc`)
   - Extracts text content while preserving document structure
   - Maintains headings, paragraphs, lists, and tables
-  - Preserves basic formatting
+  - Preserves basic formatting and styling
+
```

Comment:
Let' s collapse this into the above "Word Documents" section, and also add ".rtf